ui/page/dexclient/create_wallet_modal.go

```diff
@@ -0,0 +1,253 @@
+package dexclient
+
+import (
+	"fmt"
+	"strings"
+
+	"decred.org/dcrdex/client/asset/btc"
+	"decred.org/dcrdex/client/asset/dcr"
+	"gioui.org/layout"
+	"gioui.org/widget"
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/dcrlibwallet/dexdcr"
+	"github.com/planetdecred/godcr/dexc"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/page/components"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+const dexCreateWalletModalID = "dex_create_wallet_modal"
+
+type createWalletModal struct {
+	*load.Load
+	sourceAccountSelector *components.AccountSelector
+	modal                 *decredmaterial.Modal
+	createNewWallet       decredmaterial.Button
+	walletPassword        decredmaterial.Editor
+	appPassword           decredmaterial.Editor
+	walletName            decredmaterial.Editor
+	RPCAddress            decredmaterial.Editor
+	RPCPort               decredmaterial.Editor
+	RPCUsername           decredmaterial.Editor
+	RPCPassword           decredmaterial.Editor
+	walletInfoWidget      *walletInfoWidget
+	isSending             bool
+	walletCreated         func()
+}
+
+func newCreateWalletModal(l *load.Load, wallInfo *walletInfoWidget) *createWalletModal {
+	md := &createWalletModal{
+		Load:             l,
+		modal:            l.Theme.ModalFloatTitle(),
+		walletPassword:   l.Theme.EditorPassword(new(widget.Editor), "Wallet Password"),
+		appPassword:      l.Theme.EditorPassword(new(widget.Editor), "App Password"),
+		createNewWallet:  l.Theme.Button("Add"),
+		walletInfoWidget: wallInfo,
+		walletName:       l.Theme.Editor(new(widget.Editor), "Wallet Name"),
+		RPCAddress:       l.Theme.Editor(new(widget.Editor), "RPC Address"),
+		RPCPort:          l.Theme.Editor(new(widget.Editor), "RPC Port"),
+		RPCUsername:      l.Theme.Editor(new(widget.Editor), "RPC username"),
+		RPCPassword:      l.Theme.EditorPassword(new(widget.Editor), "RPC Password"),
+	}
+
+	md.createNewWallet.TextSize = values.TextSize12
+	md.createNewWallet.Background = l.Theme.Color.Primary
+	md.appPassword.Editor.SingleLine = true
+	md.appPassword.Editor.SetText("")
+
+	md.sourceAccountSelector = components.NewAccountSelector(md.Load).
+		Title("Sellect DCR account to pay fee").
```

Comment:
The selected account isn't only used for fee payment, it may not even be used at all for fee payment.
```suggestion
		Title("Select DCR account to use with DEX").
```